src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -262,11 +262,11 @@
                     <td id="apprentice-course" class="govuk-table__cell">@Model.CourseName</td>
                     <td class="govuk-table__cell"></td>
                 </tr>
-                @if(Model.DeliveryModel != null)
+                @if(Model.DeliveryModel.HasValue)
```

Comment:
The Delivery Model should be hidden if it's `Regular`. Most delivery models are Regular, and where this is the case most users won't even know what it means.

Reply:
Good spot!! Thanks. Updated in following commit.